package.json

```diff
@@ -57,7 +57,8 @@
     "release:minor": "yarn run release minor",
     "release:patch": "yarn run release patch",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "prepare": "husky install"
```

Comment:
Not sure if we used husky? (I avoid it in `jotai`.)

Reply:
We use husky for devtools to enforce conventional commits but we don't need the `prepare` script. So we can safely omit this.
```suggestion
```

Reply:
Hey, may I ask why you don't need the prepare script?
As I see it, the prepare script is required when a developer is pulling a fresh copy of the repo. It will then install the githooks you got in your `.husky` directory into the `.git` directory when executing `npm i` and the dev would not have to execute `npx husky install` by themselves.
And when you don't have the prepare script, a new contributor (like me) could commit with any message and no validation.

It probably doesn't / rarely matters for you as active maintainers, but I guess it could help new contributors :)

Reply:
You're absolutely right! We _do_ the `prepare` script. I was under the impression that husky auto-installs itself. It turns out [they stopped doing that long ago](https://blog.typicode.com/husky-git-hooks-autoinstall/).